client/src/js/services/DateService.js

```diff
@@ -45,6 +45,7 @@ function DateService() {
   service.current = {};
   service.next = {};
   service.util = {};
+  service.getAge = getAge;
```

Comment:
I think I have come across this before! 

I found this reference very useful http://momentjs.com/docs/#/displaying/difference/

As I understand it you can perform a query like: 
```js
/**
* returns the number of years between now (specified by moment() just like Date())
* and the specified date passed as the first parameter to the diff method
*/
moment().diff( date, 'years' );
```

It is worth noting that because we include the Angular Moment library the full moment library can be required by a controller/ service by injecting `moment` 

```js
Controller.$inject = ['moment'];

function Controller(moment) { 
  // ...
}
```

Let me know if there's anything that doesn't make sense!